libraries/common/src/main/java/androidx/media3/common/util/CodecSpecificDataUtil.java

```diff
@@ -72,6 +78,83 @@ public static boolean parseCea708InitializationData(List<byte[]> initializationD
         && initializationData.get(0)[0] == 1;
   }
 
+  /**
+   * Parses an MPEG-4 Visual configuration information, as defined in ISO/IEC14496-2
+   *
+   * @param videoSpecificConfig A byte array containing the MPEG-4 Visual configuration information
+   *     to parse.
+   * @return A pair consisting of the width and the height.
+   */
+  public static Pair<Integer, Integer> parseMpeg4VideoSpecificConfig(byte[] videoSpecificConfig) {
```

Comment:
It seems the parsing outcome is the video resolution? If so, maybe rename the method to reflect, something like 

`getVideoResolutionFromMpeg4VideoConfig()`